### PR TITLE
Use multiple values to search patients via attributes

### PR DESCRIPTION
**Pull request by enahomurphy**
# JIRA TICKET NAME:
CB-49

## SUMMARY:
Adds multiple values to search patients via attributes using "," delimiter to separate the attributes

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -9,10 +9,24 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css">
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.8.12/jquery-ui.min.js" type="text/javascript" charset="utf-8"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jquery-tagsinput/1.3.6/jquery.tagsinput.min.css">
```
Comment (dkayiwa) [inline app/index.html]: Will this work for those who do not have an internet connection?

Comment (enahomurphy) [reply app/index.html]: It won't, the libraries are been pulled from cdn. I'll work on making all the libraries offline.

File: app/index.html [hunk 2]
```diff
@@ -6,13 +6,25 @@
     <!-- openmrs favicon -->
     <link rel="shortcut icon" type="image/ico" href="../../images/openmrs-favicon.ico" />
     <link rel="icon" type="image/png" href="../../images/openmrs-favicon.png" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css">
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" href="libs/bootstrap/dist/css/bootstrap.css">
+    <script src="libs/jquery/dist/jquery.min.js" ></script>
```
Comment (dkayiwa) [inline app/index.html:10]: So where is this libs folder?

Comment (ghost) [reply app/index.html:10]: Sorry for the late reply Dkayiwa. The libs folder is managed using a frontend dependency manager called bower. It's not ideal to push external dependencies to git. When you clone this project and run npm install the libs folder automatically gets created and all the dependencies gets installed.

Comment (dkayiwa) [reply app/index.html:10]: Oh cool! So the compiled module will have the lib folder?

Comment (ghost) [reply app/index.html:10]: Yes, which contain all external dependencies.

**Pull request merged**